glidex.forms/Forms.cs

```diff
@@ -9,17 +10,19 @@ public static class Forms
 	{
 		static bool initialized;
 
+        internal static Activity GlidexActivity { get; private set; }
+
 		/// <summary>
 		/// Initializes glidex.forms, put this after your `Xamarin.Forms.Forms.Init (this, bundle);` call.
 		/// </summary>
 		/// <param name="debug">Enables debug logging. Turn this on to verify Glide is being used in your app.</param>
-		public static void Init (bool debug = false)
+		public static void Init (Activity activity, bool debug = false)
 		{
 			if (initialized)
 				return;
 
 			IsDebugEnabled = debug;
-
+            GlidexActivity = activity; 
```

Comment:
So if you follow the code here...

If the `Activity` was destroyed and recreated, then `GlidexActivity` will be holding an instance of the _old_ activity.

It won't allow for a new one to be set either:
```csharp
if (initialized)
    return;
```

Reply:
> * I'm using [Mono's (or Xamarin.Android)](https://www.mono-project.com/community/contributing/coding-guidelines/) coding style. I will get a `.editorconfig` file on master so it will format to match this repo.

I'm sorry for that, I aways use the ctrl+E+D shortcun to format my code, so this happens hehe. I'll fix that.

> * `OnDestroy`/`OnCreate` for new activities has a problem.

Maybe i'm wrong, but in Xamarin.Forms the Current Activity aways be the MainActivity. If it's not enought we can use [this approach](https://github.com/xamarin/Essentials/blob/master/Xamarin.Essentials/Platform/Platform.android.cs).

> * Not a huge fan of the name `GlidexActivity`. Maybe just `Activity` would be better?

It's ok to me,I'm not very good at naming things hehe.

Reply:
I don't think we have to go as far as make a lifecycle listener like Xamarin.Essentials did.

It looks like we can just remove this `initialized` field? It looks like it used to be here for a reason, but I don't see why it is needed anymore.

Reply:
I think it's ok to remove the `initialized`.